src/binaryen-c.h

```diff
@@ -64,13 +64,14 @@ typedef uint32_t BinaryenIndex;
 // Core types (call to get the value of each; you can cache them, they
 // never change)
```

Comment:
Please add a comment here about the new addition, that it is not a core type, but represents "no type defined / no type passed into the API" or something like that.